### PR TITLE
Distance Sensor Driver: PGA460

### PR DESCRIPTION
**Pull request by dakejahl**
This merge request is to add support to PX4 for the TI PGA460 Ultrasonic Signal Processor and Transducer Driver.
http://www.ti.com/lit/ds/symlink/pga460.pdf

**Test data / coverage**
image
https://review.px4.io/plot_app?log=961cf7f8-86ab-4d10-a912-77649316446b

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Very cool!

File: src/drivers/distance_sensor/pga460/pga460.cpp [hunk 1]
```diff
@@ -0,0 +1,851 @@
...
+		return PX4_OK;
+	} else {
+		return PX4_ERROR;
+	}
+}
+
+int pga460_main(int argc, char *argv[])
+{
+	return PGA460::main(argc, argv);
+}
```
Comment (LorenzMeier) [inline src/drivers/distance_sensor/pga460/pga460.cpp]: missing newline

**Comment by LorenzMeier** [comment]
Could you please enable this for at least one target, e.g. fmuv-5? Otherwise we don't have CI / build protection. Thanks!

**Comment by LorenzMeier** [comment]
And please not for fmu-v2, to make flash space manageable.

**Comment by dagar** [comment]
@LorenzMeier it's part of any config that pulls in the distance_sensor group (most common boards that aren't px4fmu-v2).
 - eg https://github.com/PX4/Firmware/blob/master/cmake/configs/nuttx_px4fmu-v4_default.cmake#L12

On px4fmu-v2 it's done manually per distance sensor.

**Comment by mcsauder** [comment]
We completed another 2.5hrs of flight testing against this merge request with current PX4/Firmware master branch today and with excellent results.

**Comment by dakejahl** [comment]
Is there anything else that needs to be done to get this merged? If you want to review the code feel free to suggest changes! Thanks.

File: src/drivers/distance_sensor/pga460/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,43 @@
+############################################################################
+#
+#   Copyright (c) 2016 PX4 Development Team. All rights reserved.
```
Comment (dagar) [inline src/drivers/distance_sensor/pga460/CMakeLists.txt]: date

**Comment by dakejahl** [comment]
@dagar Rebased and ready to merge.

**Pull request merged**